lib/bolt_spec/run.rb

```diff
@@ -49,6 +48,27 @@ def run_script(script, targets, arguments = nil, options = {}, config: nil, inve
       Bolt::Util.walk_keys(result, &:to_s)
     end
 
+    def apply_manifest(manifest, targets, execute: false, noop: false, config: nil, inventory: nil)
+      # The execute parameter is equivalent to the --execute option
+      if execute
```

Comment:
From a testing perspective it seems weird to design around `bolt apply ...`. If you're testing something with Bolt, it's probably a plan. But from a usable API that, it seems like a useful feature to include.

Reply:
Are you suggesting we do not implement the `execute` option? I agree that if you are not applying a manifest file the puppet code makes more sense in an apply block.

Reply:
I think what's here is ok. I'm not sure what the better default is though. Thinking about this plan-centrically the default should be `execute: true`. But I'm not really close to the people planning to use this.